.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Note: for support questions, you are strongly advised to use [Stack Overflow](https://stackoverflow.com/questions/tagged/chisel).
+-->
+**Describe the bug**
```

Comment:
can we not keep the content from our old Issue template? What about this one makes it a "new" one vs the old one?

Reply:
Okay I will remove it, I was thinking it will be relevant information for someone creating an issue.

Reply:
Okay @mwachs5, I will be waiting on your review to improve this PR, thank you.
And I also wish to ask how documentions and

Reply:
Sorry @mwachs5, I thought this was rhetorical question to follow as to why I added content from the old issue template.
The main difference between this new template and the old one is that when you click on `new issue, you get to chose between the possible issues (bug fix, feature request), and then each issue type has its own template on how to fill it.

Reply:
Sorry for replying to this late.

Reply:
sure... can we keep the basic questions the same as they currently are? At least for the issue one?

Reply:
![Screenshot from 2021-11-03 23-13-23](https://user-images.githubusercontent.com/81782853/140267204-4e71337b-8608-4a76-b77c-137edc29f7f5.png)


This is the window that will pop up when you click the `new issue` button,
from there after choosing your issue type,

Reply:
Yes we can.